configs/mmpose/pose-detection_rtmo_onnxruntime_dynamic.py

```diff
@@ -0,0 +1,25 @@
+_base_ = ['./pose-detection_static.py', '../_base_/backends/onnxruntime.py']
```

Comment:
have we released the model config and ckpt for RTMO model?

Reply:
The models will be released tonight or tomorrow

Reply:
The models have been released at https://github.com/open-mmlab/mmpose/tree/dev-1.x/configs/body_2d_keypoint/rtmo